### ISSUE TITLE
Fenix private mode doesn't trigger Gboard private mode.

### ISSUE THREAD

**Issue by Lululalu** [mozilla-mobile/fenix#489]

**Comment by pocmo** [comment]
It will after #347. Currently private mode `Session`s are regular `Session`s.

**Comment by pocmo** [comment]
@sblatz Do you want to close this as a duplicate of #347 or do you want to keep the bug open to verify this after #347 is closed?

**Comment by sblatz** [comment]
Made a note of this in #347!
